app/js/fm-player/controllers/PlayerCtrl.js

```diff
@@ -80,15 +80,22 @@ angular.module("sn.fm.player").controller("PlayerCtrl", [
         };
 
         /**
-         * Increment volume by 10
+         * Value to increment/decrement volume by
+         * @property volumeStep
+         * @type     {Number}
+         */
+        $scope.volumeStep = 5;
+
+        /**
+         * Increment volume
          * @method volumeUp
          */
         $scope.volumeUp = function volumeUp() {
             PlayerVolumeResource.get({}, function(volume){
 
-                // round volume to nearest 10
-                volume.volume = Math.round(volume.volume / 10) * 10;
-                volume.volume = volume.volume + 10;
+                // round volume
+                volume.volume = Math.round(volume.volume / $scope.volumeStep) * $scope.volumeStep;
+                volume.volume = volume.volume + $scope.volumeStep;
 
```

Comment:
The division then multiplication is to ensure it is a multiple of the step. So if the volume somehow was 52 `((52 / 5) * 5) + 5` would round this to the nearest five then increment by 5, resultign volume would be 55. I suppose I don't actually need `Math.round()` in this case.